### PR TITLE
Feature - TRSV2-407 - ROI deficiency task list 

### PR DESCRIPTION
**Pull request by chris-pettinga**

### CODE REVIEW

File: trade_remedies_public/templates/cases/submissions/application/submit.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-    });
-</script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        window.dataLayer.push({
+            'event': 'apply_for_case',
+            'user_id': '{{user.id}}',
+            'case_id': '{{case_id}}'
+        });
+    </script>
```
Comment (yusufsheiqh) [inline trade_remedies_public/templates/cases/submissions/application/submit.html:12]: Is this something that the `djhtml` picked up?

File: trade_remedies_public/templates/cases/submissions/interest/submit.html [hunk 2]
```diff
@@ -1,28 +1,12 @@
...
-    <script>
-        window.dataLayer = window.dataLayer || [];
-        window.dataLayer.push({
-            'event': 'register_interest',
-            "user_id": '{{user.id}}',
-            "client_type": "{{case.type}}",
-            "case_id": "{{case_id}}"
-        });
-    </script>
-{% endif %}
```
Comment (yusufsheiqh) [inline trade_remedies_public/templates/cases/submissions/interest/submit.html:12]: Seems like we are losing this here... is that intentional?

File: trade_remedies_public/templates/cases/submissions/application/submit.html (hunk 1 above)
Comment (chris-pettinga) [reply trade_remedies_public/templates/cases/submissions/application/submit.html:12]: Yes it is being indented further as it is in a django templatetag

File: trade_remedies_public/templates/cases/submissions/interest/submit.html (hunk 2 above)
Comment (chris-pettinga) [reply trade_remedies_public/templates/cases/submissions/interest/submit.html:12]: yes as we are adding it in v2 ROI journey instead, good spot.

**Review by yusufsheiqh — APPROVED** [review]

**Pull request merged**